src/main/java/ru/skypro/homework/service/impl/AuthServiceImpl.java

```diff
@@ -1,46 +1,78 @@
 package ru.skypro.homework.service.impl;
 
-import org.springframework.security.core.userdetails.User;
-import org.springframework.security.core.userdetails.UserDetails;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.security.provisioning.UserDetailsManager;
 import org.springframework.stereotype.Service;
+import ru.skypro.homework.config.MyUserDetails;
+import ru.skypro.homework.config.MyUserDetailsService;
+import ru.skypro.homework.constants.Constants;
+import ru.skypro.homework.dto.Login;
 import ru.skypro.homework.dto.Register;
+import ru.skypro.homework.entity.UserEntity;
+import ru.skypro.homework.mapper.RegisterMapper;
+import ru.skypro.homework.repository.UserRepo;
 import ru.skypro.homework.service.AuthService;
+import ru.skypro.homework.check.CheckService;
+import java.util.Optional;
 
+@Slf4j
 @Service
+@AllArgsConstructor
 public class AuthServiceImpl implements AuthService {
 
-    private final UserDetailsManager manager;
+    private final MyUserDetailsService myUserDetailService;
```

Comment:
```suggestion
    private final UserDetailsService myUserDetailService;
```
инъекцию по интерфейсу делать нужно